charts/management-portal/templates/secrets-postgres-keystore.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "management-portal.fullname" . }}-root-cert
+  labels:
+    app: {{ template "management-portal.name" . }}
+    chart: {{ template "management-portal.chart" . }}
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
+type: Opaque
+data:
+  root.crt: {{ .Files.Get "files/BaltimoreCyberTrustRoot.crt.pem" | b64enc | quote }}
```

Comment:
For more general application, change this to `files/root.crt`.

Reply:
Fixed in fbda1ed